### ISSUE TITLE
为什么paddle.nn中的类不同通过mmcv中Registry类实例化

### ISSUE THREAD

**Issue by zgq91** [PaddlePaddle/Paddle#40075]
paddle2.2中 通过mmcv中Registry注册实例化nn.BatchNorm2d时，会报错AttributeError: module 'paddle.nn' has no attribute 'BatchNorm2d'

**Comment by gongel** [comment]
能提供一下代码吗？

**Comment by zgq91** [comment]
> 能提供一下代码吗？
from mmcv.utils import Registry

NORM_LAYERS = Registry('norm layer')
NORM_LAYERS.register_module('BN2d', module=nn.BatchNorm2d)

**Comment by MissPenguin** [comment]
mmcv是torch的呀

**Comment by zgq91** [comment]
> mmcv是torch的呀

我知道呀，但它注册类是python代码，不包含torch，它只是通过注册的方式来实例化对象

**Comment by gongel** [comment]
Hi, 可以去mmcv那边提个issue看看吗？
